dry-core.gemspec

```diff
@@ -30,6 +30,7 @@ Gem::Specification.new do |spec|
 
   # to update dependencies edit project.yml
   spec.add_runtime_dependency "concurrent-ruby", "~> 1.0"
+  spec.add_runtime_dependency "zeitwerk", "~> 1.4"
```

Comment:
this must be here https://github.com/dry-rb/dry-core/blob/250ad54dfa6bc1277c4af64bdccbe045d96d04bd/project.yml#L11

Reply:
funny that I even I forgot 😅 thanks! I'll be fixing all of that in main branches.